https://sourceforge.net/p/x3270/code/merge-requests/2/

commit f86253bc15060db29063c186b05dc9bbaa38593a
Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Sun Jun 18 19:49:43 2017 +0200

    allow to override build date
    
    to allow reproducible builds of x3270
    
    See https://reproducible-builds.org/ for why this is good
    and https://reproducible-builds.org/specs/source-date-epoch/ for the definition of this variable.
    
    Note: this patch variant only works with GNU date
    
    leftover after incomplete merge in 7591b303beefed762c24386214beca432024c785

diff --git a/Common/mkversion.sh b/Common/mkversion.sh
index ad1c1067..4da3620a 100755
--- a/Common/mkversion.sh
+++ b/Common/mkversion.sh
@@ -50,7 +50,7 @@ user=${LOGNAME-$USER}
 # rpq.c will return this string of numbers in bcd format
 # It is OK to change the length (+ or -), but use
 # decimal (0-9) digits only. Length must be even number of digits.
-rpq_timestamp=`date +%Y%m%d%H%M%S`
+rpq_timestamp=`$date +%Y%m%d%H%M%S`
 
 app=${1-x3270}
 cat <<EOF
